src/controller/python/test_client/Dockerfile

```diff
@@ -11,7 +10,6 @@ RUN apt-get update
 
 # Install Dependencies for setting up CHIP environment
 RUN apt-get install -y apt-utils \
-                       git \
```

Comment:
why remove git install? Won't it just be skipped over? Or does it cause issues?

Reply:
Not needed anymore, because this docker file originally build the chip sdk inside the container so we needed to install git so we could clone and checkout the sdk